api/src/middleware/__tests__/assessmentsRouter.ts

```diff
@@ -59,17 +81,520 @@ describe('assessmentsRouter', () => {
   });
 
   describe('GET /:assessmentId/submissions/:submissionId', () => {
-    it('should return the submission information', done => {
-      const response = {
-        behaviour:
-          'Returns the submission information (metadata, answers, etc)',
+    it("should show a facilitator the full submission information for a participant's ungraded submitted assessment, including the correct answers", done => {
+      const facilitatorPrincipalId = 3;
+      const participantPrincipalId = 2;
+      const programAssessmentId = 1;
+      const submissionId = 1;
+      const curriculumAssessmentId = 1;
+      const curriculumAssessment = {
```

Comment:
You've got a lot of data that you're sharing that is very similar or exactly similar between all your tests. I'd like you to move that shared data into a space between L83 and L84 and reference them as needed. That way we don't have any repetition in the file that isn't absolutely necessary.

For an example of how we've done this before, please reference `api/src/services/__tests__/programsService.ts`.